configureWorkspace/configDebugProvider.ts

```diff
@@ -9,20 +9,14 @@ import * as vscode from 'vscode';
 export class DockerDebugConfigProvider implements vscode.DebugConfigurationProvider {
 
     public provideDebugConfigurations(folder: vscode.WorkspaceFolder | undefined, token?: vscode.CancellationToken): vscode.ProviderResult<vscode.DebugConfiguration[]> {
-
-        const config: vscode.DebugConfiguration = {
-            name: 'Docker: Attach to Node',
-            type: 'node',
-            request: 'attach',
-            port: 9229,
-            address: 'localhost',
-            localRoot: '\${workspaceFolder}',
-            remoteRoot: '/usr/src/app',
-            protocol: 'inspector'
-        };
-
-        return [config];
-
+        return vscode.window.showInputBox({ value: '/usr/src/app', prompt: 'Please enter your Docker remote root'}).then(remoteRoot => {
+            return [{
+                name: 'Docker: Attach to Node',
+                type: 'node',
+                request: 'attach',
+                remoteRoot
+            }];
+        });
```

Comment:
For any new code added, we tend to avoid thenable in favor of async-await. 

Debugging this code has seemed easier when the code is written as : 
```
let remoteRoot = await vscode.window.showInputBox({ value: '/usr/src/app', prompt: 'Please enter your Docker remote root'}); 
return [{
name: 'Docker: Attach to Node',
type: 'node',
request: 'attach',
remoteRoot
}]; 
```

You would need to make the function async however, so I'll defer to @StephenWeatherford  whether requiring this standard is worth it here. :)

Reply:
Yes, please do convert.  Also, please use ext.ui.showInputBox instead of vscode.window.showInputBox. Note that this will throw on user cancel, which I assume makes sense here?

Reply:
Thanks for the review and suggestions I have pushed a commit which takes care of this.